### ISSUE TITLE
[Feature]: Update interface of the home screen

### ISSUE THREAD

**Issue by jvondermarck** [jvondermarck/dinosaur-exploder#77]
### 👉 Please confirm the following

- [X] I agree to follow the [Code of Conduct](https://github.com/dylan-power/dinosaur-exploder/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I checked the [current issues](https://github.com/dylan-power/dinosaur-exploder/issues) for duplicate features.

### 😍 Describe the solution you'd like

About the home screen of the game it needs to have some update of the interface : 
- Try to stylized the two buttons
![image](https://user-images.githubusercontent.com/62793491/178150207-dc40e415-783e-4819-88fd-a0946d1ce9ea.png)

- Change the black background to a space background with planets, maybe a moving background in the space ??
![image](https://user-images.githubusercontent.com/62793491/178150244-4dbbc641-3432-4a7c-acfa-f2bc8963be3d.png)

- Add somewhere the icon of the game (download this specific [image](https://pixabay.com/ro/illustrations/dinozaur-animal-autocolant-reptila-6072475/))

**Comment by RamonASV** [comment]
Hello, I currently made some changes while I was working on #76  after I got stuck.
<img width="549" alt="Screen Shot 2022-07-11 at 18 37 25" src="https://user-images.githubusercontent.com/104037313/178376242-0941beae-27c3-43e9-afc6-8945a15800d7.png">
I'm still checking on how to include a moving background, but if it's okay I can upload these changes

**Comment by jvondermarck** [comment]
Hii, @RamonASV well done! Can you just try to put some style to the music button, by removing the white background, because right now it doesn't fit really well to the rest of the design? Otherwise everything looks good.
I'm gonna check your error on the other issue ASAP. 🤔

**Comment by jvondermarck** [comment]
EDIT 18 April 2024 :

The home screen still needs to be updated in a better way, mainly align the buttons and change their styles.
![image](https://github.com/dylan-power/dinosaur-exploder/assets/62793491/7b3e03ef-41c4-41fa-abfd-2c2daa8db104)

**Comment by ThreeWater1037** [comment]
Hello, I am a beginner on GitHub and currently learning Java and JavaFX😊. May I try to work on this issue?

**Comment by jvondermarck** [comment]
Yes I will assign you on this issue, thanks for your contribution !! :)

**Comment by ThreeWater1037** [comment]
Hello, I have made some preliminary modifications to the main menu style.
I replaced the "Start Game" and "Quit Game" buttons with rounded buttons and adjusted their colors to better match the theme. 
Additionally, I added a background image—the one featuring the cosmos—and set it to scroll horizontally. If you find these design changes acceptable, I will proceed to submit my PR😊.

https://github.com/user-attachments/assets/edd79d56-d7fb-4bc1-ac4b-3161f607ac53
